src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -319,7 +322,12 @@ def get(self, **kwargs):
         result = self._intersection_of_request_info(request_info)
         if len(result) == 0:
             return []
-        return rows([result])
+        
+        if common_dict == 1:
+            response_list = result.values()
```

Comment:
Does result values guaranteed to have identical structure? What about request names, they will not appear in result.values() since they're the keys of result dict